src/ConfigPage/ConfigPage.tsx

```diff
@@ -40,14 +40,14 @@ const ConfigPage: FC<ConfigPageProps> = ({ api, defaultSecret = '', defaultToken
 				container
 				alignItems="center"
 				justify="center"
-				style={{ margin: '0 0 1rem 0' }}
+				className="api-container"
 			>
 				<Grid item>API</Grid>
-				<Grid item style={{ margin: '0 1rem 0 1rem' }}>
+				<Grid item >
```

Comment:
There's an extra space here

```suggestion
				<Grid item>
```